### PR TITLE
Restyle Application Thumbnails & condense thumbnail macros

### PR DESCRIPTION
**Pull request by JDiPierro**
FIRST! :P

On displays large enough this restyles the application cards to be horizontal. I feel this is a better browsing experience when the display size allows for it. On small displays such as mobile devices the cards should look the same as they do now.

**Desktop:**
image

**Mobile:**
image

### CODE REVIEW

File: puffin/static/styles/puffin.css [hunk 1]
```diff
@@ -34,14 +34,6 @@ body {
 }
 
 /* Application thumbnails */
-.app.thumbnail {
-    height: 335px;
-}
```
Comment (loomchild) [inline puffin/static/styles/puffin.css]: Actually I prefer to keep the constant size if some of images does not display or text is too short.

File: puffin/static/styles/puffin.css [hunk 2]
```diff
@@ -34,14 +34,6 @@ body {
...
-
-.app.thumbnail a {
-    color: black;
-}
```
Comment (loomchild) [inline puffin/static/styles/puffin.css]: I'd prefer to keep the title links black, what do you think?

File: puffin/templates/macros.html [hunk 3]
```diff
@@ -1,23 +1,28 @@
+{% import "bootstrap/wtf.html" as wtf %}
+
 {% macro application_thumbnail(application) %}
-    <div class="col-sm-4 col-md-3">
+    <div class="col-sm-12 col-md-6">
     <div class="app thumbnail">
```
Comment (loomchild) [inline puffin/templates/macros.html]: OK, good idea to merge two types of thumbnails. You forgot a "new" class - I will add it.

File: puffin/static/styles/puffin.css (hunk 1 above)
Comment (loomchild) [reply puffin/static/styles/puffin.css]: Ah, it messes up the mobile display, OK, let's drop it for simplicity.

File: puffin/static/styles/puffin.css (hunk 2 above)
Comment (JDiPierro) [reply puffin/static/styles/puffin.css]: Oops, yeah. I'm not sure why I removed that.

**Pull request merged**